ci-operator/config/openshift/special-resource-operator/openshift-special-resource-operator-release-4.10.yaml

```diff
@@ -36,19 +62,109 @@ releases:
 resources:
   '*':
     limits:
-      memory: 4Gi
+      memory: 8Gi
     requests:
-      cpu: 100m
-      memory: 200Mi
+      cpu: 300m
+      memory: 500Mi
 tests:
-- as: unit
-  commands: make test
```

Comment:
There is no such Makefile target. This test passed until now because of "test emptyness".